enterprise_access/apps/api/v1/views.py

```diff
@@ -925,3 +925,26 @@ def redeem(self, request, *args, **kwargs):
         finally:
             if lock_acquired:
                 policy.release_lock(learner_id, content_key)
+
+    @action(detail=False, methods=['get'])
+    def redemption(self, request, *args, **kwargs):
+        """
+        Return redemption records for given `group_id`, `learner_id` and `content_key`
+        """
+        serializer = serializers.SubsidyAccessPolicyRedeemListSerializer(data=request.query_params)
+        serializer.is_valid(raise_exception=True)
+
+        group_uuid = serializer.data['group_id']
+        learner_id = serializer.data['learner_id']
+        content_key = serializer.data['content_key']
+
+        redemptions = []
+        policies = SubsidyAccessPolicy.objects.filter(group_uuid=group_uuid)
+        for policy in policies:
+            redemption = policy.has_redeemed(learner_id, content_key)
+            redemptions.append(redemption)
+
+        return Response(
+            redemptions,
+            status=status.HTTP_200_OK,
+        )
```

Comment:
I think we want to run `redemptions` through a serializer before returning it, right?

Reply:
redemptions is a list of responses where each response is the return value of `policy.has_redeemed` received from enterprise-subsidy. This is not finalized yet due to unavailability of communication between enterprise-access and enterprise-subsidy. This whole viewset will be revisited once all the endpoints that are needed will be implemented.

Reply:
Ah, ok, makes sense.